src/auto-evo/simulation/food_source/EnvironmentalFoodSource.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Linq;
+
+public class EnvironmentalFoodSource : IFoodSource
+{
+    private readonly Compound compound;
+    private readonly Compound glucose = SimulationParameters.Instance.GetCompound("glucose");
+
+    private BiomeConditions biomeConditions;
+    private float totalEnvironmentalEnergySource;
+
+    public EnvironmentalFoodSource(Patch patch, string compound, float constant)
+    {
+        biomeConditions = patch.Biome;
+        this.compound = SimulationParameters.Instance.GetCompound(compound);
+        totalEnvironmentalEnergySource = patch.Biome.Compounds[this.compound].Dissolved * constant;
+    }
+
+    public float FitnessScore(Species species)
+    {
+        var microbeSpecies = (MicrobeSpecies)species;
+
+        var photosynthesisingScore = 0.0f;
```

Comment:
This variable name is much more specific than the one of the class (environmental source), so you may want to change this.